protos/telemetry_server/telemetry_server.proto

```diff
@@ -188,6 +242,35 @@ message PublishUnixEpochTimeResponse {
     TelemetryServerResult telemetry_server_result = 1;
 }
 
+message ArmDisarmResponse
+{
+    TelemetryServerResult telemetry_server_result = 1;
+    ArmDisarm arm = 2;
```

Comment:
Why is this required as well when it already returns whether the arm or disarm command succeeded or not?

Reply:
You cannot tell if it is an "Arm" or "Disarm" event